templates/modelExpt/run_modelComp_audapter.m

```diff
@@ -0,0 +1,217 @@
+function [expt] = run_modelComp_audapter(expt)
+% Companion function for modelComp (based on cerebAAF (based on hearingLossAAF))
+% Don't run this directly -- run `run_modelComp_expt` instead.
+
+if nargin < 1
+    error('Must pass in valid expt variable from run_modelComp_expt')
+end
+
+%%
+
+if ~exist(expt.dataPath,'dir')
+    mkdir(expt.dataPath)
+end
+
+if isfield(expt,'dataPath')
+    outputdir = expt.dataPath;
+else
+    warning('Setting output directory to current directory: %s\n',pwd);
+    outputdir = pwd;
+end
+
+% assign folder for saving trial data
+% create output directory if it doesn't exist
+trialdirname = 'temp_trials';
+trialdir = fullfile(outputdir,trialdirname);
+if ~exist(trialdir,'dir')
+    mkdir(outputdir,trialdirname)
+end
+
+%% set up stimuli
+stimtxtsize = 200;
+
+firstTrial = expt.startTrial; % defaults to 1
+lastTrial = expt.ntrials;
+
+%% set up audapter
+audioInterfaceName = 'Focusrite USB'; %SMNG default for Windows 10
+Audapter('deviceName', audioInterfaceName);
+
+% set files for vowel tracking
+% ostFN = fullfile(get_exptRunpath(expt.trackingFileLoc),[expt.trackingFileName 'Working.ost']);
+% pcfFN = fullfile(get_exptRunpath(expt.trackingFileLoc),[expt.trackingFileName 'Working.pcf']);
+% check_file(ostFN);
+% check_file(pcfFN);
```

Comment:
Remove if we're not using OSTs in this model experiment; could create one in which we do use them